backend/routes.py

```
@@ -37,59 +38,68 @@ def register_event_handlers():
+                raw_data=payload
+            )
+            await event_bus.dispatch(event)
+        else:
+            logger.info(f"No matching event type for header: {event_header} with action: {action}")
 
-    return {"status": "ok"}
+        return {"status": "ok"}
+
+    except HTTPException:
+        raise
+    except Exception as e:
```

Ruff (0.15.2)

[error] Local variable `e` is assigned to but never used

Remove assignment to unused variable `e`

(F841)